src/osip-monitoring-view/src/main/java/edu/kit/pse/osip/monitoring/view/dashboard/ProgressOverview.java

```diff
@@ -62,6 +62,7 @@ protected ProgressOverview(AbstractTank tank) {
      */
     protected final void setFillLevelProgressEnabled(boolean progressEnabled) {
         fillLevel.setProgressEnabled(progressEnabled);
+        tabFillLevel.setDisable(!progressEnabled);
```

Comment:
Ich würde hier die Tabs nicht ausschalten. Die Charts finde ich in Ordnung, aber es ist komisch, wenn ich einen deaktivierten Tab habe, auf einen anderen schalten und nicht wieder zurückschalten kann.